api/openstorage.yaml

```diff
@@ -173,23 +576,255 @@ paths:
+                      "passphrase": "",
+                      "snapshot_schedule": "",
+                      "scale": 0,
+                      "sticky": false,
+                      "max_backups": 0,
+                      "backup_schedule": ""
+                    },
+                    "usage": "254148608",
+                    "last_scan": "2017-03-30T02:02:35Z",
+                    "format": "ext4",
+                    "status": "up",
+                    "state": "attached",
+                    "attached_on": "e612618a-3c8f-490d-b288-674089de7254",
+                    "device_path": "/dev/pxd/pxd556970537543779086",
+                    "attach_path": [
+                      "/var/lib/osd/mounts/mysql_volume"
+                    ],
+                    "replica_sets": [
+                      {
+                        "nodes": [
+                          "e612618a-3c8f-490d-b288-674089de7254"
+                        ]
+                      }
+                    ],
+                    "error": "",
+                    "runtime_state": [
+                      {
+                        "runtime_state": {
+                          "FullResyncBlocks": "[{1 0} {-1 0} {-1 0} {-1 0} {-1 0}]",
+                          "ID": "0",
+                          "ReadQuorum": "1",
+                          "ReadSet": "[1]",
+                          "ReplicaSetCurr": "[1]",
+                          "ReplicaSetNext": "[1]",
+                          "ResyncBlocks": "[{1 0} {-1 0} {-1 0} {-1 0} {-1 0}]",
+                          "RuntimeState": "clean",
+                          "TimestampBlocksPerNode": "[0 0 0 0 0]",
+                          "TimestampBlocksTotal": "0",
+                          "WriteQuorum": "1",
+                          "WriteSet": "[1]"
+                        }
+                      }
+                    ],
+                    "secure_device_path": "",
+                    "background_processing": false
+                  }
+                ]
     post:
-      summary: Create New Volume
-      description: Create new volume according to spec
-      parameters:
-        - name: spec
-          in: body
-          description: Volume spec
-          required: true
-          schema:
-            $ref: '#/definitions/VolumeCreateRequest'
-      tags:
-        - Volumes
-      responses:
-        '200':
-          description: UUID of neewly created volume
-          schema:
-            $ref: '#/definitions/VolumeCreateResponse'
+        summary: Create New Volume
+        description: Create new volume according to spec
+        parameters:
+          - name: spec
+            in: body
+            description: |
+             Volume create path example:
+             POST /v1/osd-volumes
+              Volume spec example:
+              ``` json
+                  "locator": {
+                     "name": "cassandra-vol",
+                     "labels":{
+                        SLA: "Gold"
```

Comment:
Yes, it should be in Quotes. Fixed it as well 

{
    "guid": "",
    "name": "testsla",
    "blockSize": 65536,
    "snapshotInterval": 60,
    "dedupe": 1,
    "cos": 1,
    "format": "ext4",
    "labels": {
        "SLA": "Gold"
    },
    "halevel": 3,
    "size": 21474836480,
    "volumegroup": "",
    "shared": false,
    "readonly": false,
    "source": "",
    "copies": 1
}